### PR TITLE
Bit of work

### PR DESCRIPTION
**Pull request by ProfessorProspector**

### CODE REVIEW

File: mappings/rnet/minecraft/chunk/ChunkGeneratorFlat.mapping [hunk 1]
```diff
@@ -1,3 +1,3 @@
-CLASS bsc
+CLASS bsc rnet/minecraft/chunk/ChunkGeneratorFlat
```
Comment (thiakil) [inline mappings/rnet/minecraft/chunk/ChunkGeneratorFlat.mapping]: stray `r` there

**Comment by gegy1000** [comment]
Once we're happy with both these PRs I'll merge them locally with a custom tool which is easier for mappings.

File: mappings/net/minecraft/client/render/entity/EntityRendererPufferfish.mapping [hunk 2]
```diff
@@ -1,3 +1,8 @@
 CLASS dcn net/minecraft/client/render/entity/EntityRendererPufferfish
+	FIELD j puffState I
+	FIELD k MODEL_0 Lcsa;
```
Comment (gegy1000) [inline mappings/net/minecraft/client/render/entity/EntityRendererPufferfish.mapping]: Maybe more descriptive names would be appropriate? Like `MODEL_PUFFED` for the last one.

File: mappings/net/minecraft/client/render/entity/model/EntityModelPufferfishState0.mapping [hunk 3]
```diff
@@ -0,0 +1,3 @@
+CLASS csa net/minecraft/client/render/entity/model/EntityModelPufferfishState0
```
Comment (gegy1000) [inline mappings/net/minecraft/client/render/entity/model/EntityModelPufferfishState0.mapping]: Sample applies to these class names as with the model fields.

File: mappings/net/minecraft/client/render/entity/EntityRendererPufferfish.mapping (hunk 2 above)
Comment (ProfessorProspector) [reply mappings/net/minecraft/client/render/entity/EntityRendererPufferfish.mapping]: Yeah, honestly I couldn't really figure out which one is which.

Comment (gegy1000) [reply mappings/net/minecraft/client/render/entity/EntityRendererPufferfish.mapping]: They're probably ordered by size.

**Comment by gegy1000** [comment]
Merged (with custom tool) into #10 for ease.

**Pull request closed without merging**